src/utils/resolve.js

```diff
@@ -15,6 +16,10 @@ export default function resolve(css, styles) {
     inlineStyles,
   } = separateStyles(flattenedStyles);
 
+  aphroditeStyles.forEach((stylesObj) => {
+    if (stylesObj.noRTL) stylesObj._definition = stylesObj.noRTL;
```

Comment:
Good edge case! Please add a test to cover calling resolve and resolveWithRTL one after the other.